apps/fiona/fiona.cpp

```diff
@@ -81,9 +81,15 @@
     // Thus, we do not enable it in this case.
         #define _GLIBCXX_PARALLEL
     #endif  // #if !defined(__MINGW32__) || defined(__amd64__) || defined(__x86_64__) || defined(__ia64__)
+
+    #if defined(__INTEL_COMPILER)
+        #pragma message("The Intel Compiler crashes with _GLIBCXX_PARALLEL defined (at least until v16.0.2). Therefore, the feature will be disabled.")
```

Comment:
No, because on update 16.0.3 I wouldn't be notified.

If they fix it in 16.0.3 then yes. Because then it is only a hint for older compiler versions.

For the meantime it should always notifiy me.